esp/scm/ws_workunits.ecm

```diff
@@ -2204,12 +2205,25 @@ ESPstruct [nil_remove] WUResponseProperty
     [optional] string CreatorType;  // What type of engine created it.
 };
 
+ESPstruct [nil_remove] WUResponseNote
+{
+    [optional] string Source;       // Thor, Eclcc, Analyser etc
+    [optional] string Message;
+    [optional] int64 ErrorCode;
```

Comment:
Is this really an int64?

Reply:
(JS Can't handle a full int64...)

Reply:
The error code is 32bit unsigned - I'll change the type.